specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json

```diff
@@ -2974,6 +2974,12 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
```

Comment:
@hkuadithya please scope this change to the 2020-11-01-preview API only.

Reply:
@jaysterp I'm assuming you want me to revert changes made to the `2019-06-01-preview/containerregistry_build.json` only.
Is that correct?

Reply:
Please ensure the change in this PR is only for 2020-11-01-preview as this is the new API version being currently reviewed in the swagger PR related to this branch. Please include the systemData changes for the older API versions in a separate PR.

Reply:
@jaysterp Done. Thanks for your patience :)